src/Microsoft.AspNetCore.Http.Features/Microsoft.AspNetCore.Http.Features.csproj

```diff
@@ -8,7 +8,6 @@
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore</PackageTags>
-    <EnableApiCheck>false</EnableApiCheck>
```

Comment:
Wondering whether this deletion and the one in should be commited separately to make it easier to see c9f12a2 was reverted. Thoughts?